src/python/WMCore/WMBS/Job.py

```diff
@@ -222,20 +222,6 @@ def getFiles(self, type="list"):
         self.commitTransaction(existingTransaction)
         return WMJob.getFiles(self, type)
 
-    def getFileLocations(self):
```

Comment:
@hufnagel @vytjan folks, is this method used by T0 or by any of your scripts?

Reply:
@amaltaro I don't think it's used anywhere by T0, but I'd rather wait for Dirk to confirm this.

Reply:
I am on vacation. If you want to know, "grep -r ", on the Tier code :-).

Reply:
I wasn't talking about just this method but I grepped for T0 code. I was just wondering whether there is the ops specific code (not in git repository) using one of the DAOs.
If no one knows, I guess Not. 😄

Reply:
All code I know about is in the github repo. Tier0Ops would have to comment on scripts they use.